bench/irmin-pack/tree.ml

```diff
@@ -409,20 +816,28 @@ module Bench_suite (Store : STORE) = struct
         config.ncommits config.nchain_trees config.depth repo_pp
         Benchmark.pp_results result
 
-  let run_read_trace config =
-    reset_stats ();
-    let commits, n =
-      Parse_trace.populate_array config.ncommits_trace config.flatten
+  let run_read_trace config stats =
+    let commit_seq =
+      Bootstrap_trace.open_commit_sequence config.ncommits_trace config.flatten
         config.commit_data_file
     in
-    let config = { config with ncommits_trace = n } in
-    let* repo, on_commit, repo_pp = Store.create_repo n config in
-    let* result =
-      Trees_trace.add_commits repo commits on_commit |> Benchmark.run config
+    let* repo, on_commit, on_end, repo_pp = Store.create_repo config in
+
+    let t0_cpu = Sys.time () in
+    let t0 = Mtime_clock.counter () in
```

Comment:
Just curious, what is the reason to measure both cpu and overall time?

Reply:
I'm just curious about the differences those 2 timers show, no specific use yet